photo-app/app/assets/javascripts/credit_card_form.js

```diff
@@ -0,0 +1,58 @@
+let show_error, submitHandler;
+
+show_error = function (message) {
```

Comment:
```suggestion
const show_error = function (message) {
```
you can just do this right?